android/BOINC/app/src/main/java/edu/berkeley/boinc/attach/AboutActivity.kt

```diff
@@ -0,0 +1,35 @@
+package edu.berkeley.boinc.attach
+
+import android.content.pm.PackageManager
+import android.content.pm.PackageManager.PackageInfoFlags
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import edu.berkeley.boinc.R
+import edu.berkeley.boinc.utils.Logging
+
+class AboutActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_about)
+        // about_title is used in AndroidManifest.xml but the compiler doesn't detect it
+        // So we use it here to keep it happy
+        val _about_title = R.string.about_title
```

Comment:
This doesn't look correct to me

Reply:
New commit removes this in order to see if gradle complains about it again or if I was just too tired the first time around

Reply:
If Gradle complains - that could be an issue in Gradle script as well ;)